### PR TITLE
data: add high quality loading screens and images

### LINKED ISSUE #358: Feature request: add loading screens like the PSX version.
Is it possible to add the loading screens from PSX version to Tomb1Main?

The Tomb2Main project already did it and it looks great.

### PR DESCRIPTION
**Pull request by walkawayy**
Resolves #358.

#### Checklist

- [X] I have read the [coding conventions](https://github.com/LostArtefacts/TR1X/blob/master/CONTRIBUTING.md#coding-conventions)
- [X] I have added a changelog entry about what my pull request accomplishes, or it is an internal change

#### Description
Added an option to use PS1 loading screens. Added high quality images for the Eidos, Unfinished Business title, Unfinished Business credit, and final statistics screens. Moved all images to `data/images`.

Loading screens function identically to the display picture gameflow, except they can be disabled via the config tool. I put it in the `Gameplay options` tab because that's where the Eidos skip option is, but maybe both should be in `UI`?

Lastly, would @ajtudela be willing to provide a Spanish config tool translation? If you wish to maintain git `credit` I think you should be able to push a new commit to this branch? Or we can we co-author a commit? Idk too much about that though, so maybe rr can chime in. Ste was helpful and provided a French translation. Maybe we could add both of them to the credits in the README?

### CODE REVIEW

File: bin/cfg/TR1X_gameflow.json5 [hunk 1]
```diff
@@ -152,6 +157,7 @@
             ],
             "sequence": [
                 {"type": "play_fmv", "fmv_path": "fmv/lift.avi"},
+                {"type": "loading_screen", "picture_path": "data/images/greece_saturn.png", "display_time": 5},
```
Comment (lahm86) [inline bin/cfg/TR1X_gameflow.json5]: Is it intended that Folly has the Saturn image while the rest of Greece doesn't?

File: bin/cfg/TR1X_gameflow.json5 [hunk 2]
```diff
@@ -531,7 +547,7 @@
             "music": 2,
             "inherit_injections": false,
             "sequence": [
-                {"type": "display_picture", "picture_path": "data/eidospc.png", "display_time": 1},
+                {"type": "display_picture", "picture_path": "data/eidos.png", "display_time": 1},
```
Comment (lahm86) [inline bin/cfg/TR1X_gameflow.json5]: Should this be `data/images/eidos.png`?

**Review by lahm86 — COMMENTED** [review]
A couple of queries, otherwise LGTM.
I think having this in `Gameplay options` in the config tool is fine too.

**Review by rr- — COMMENTED** [review]
LGTM. Waiting for the credits change mentioned in Discord, eidos.png fix and Spanish translation.

**Review by rr- — COMMENTED** [review]
Since we bundle credits_3_alt.png that is unused, can we also bundle the hi-res more OG-looking title screen and keep it unused?

File: bin/cfg/TR1X_gameflow.json5 (hunk 1 above)
Comment (walkawayy) [reply bin/cfg/TR1X_gameflow.json5]: Oh yes I forgot to ask about this part. POSIX suggested using this loading screen once for St Francis Folly as a Saturn Easter egg. I could also leave it as an alternative in the repo like the PS1 credits. @rr-

**Comment by ajtudela** [comment]
Yes, np. I'll push the new commit with the spanish translation to this branch in a few hours

File: bin/cfg/TR1X_gameflow.json5 (hunk 1 above)
Comment (rr-) [reply bin/cfg/TR1X_gameflow.json5]: Yes let's go with PS1 version and leave Saturn as an easter egg.

**Comment by walkawayy** [comment]
Alright pushed all my remaining changes. We just need the more OG-like alt title screen optimized and the Spanish config translation. Thanks for the assists!

**Comment by walkawayy** [comment]
@ajtudela are you still able to add the Spanish translation? Apologies if you are busy and just haven't got to it yet.

**Comment by ajtudela** [comment]
> @ajtudela are you still able to add the Spanish translation? Apologies if you are busy and just haven't got to it yet.

I made a PR to the branch: https://github.com/walkawayy/TR1X/pull/1

Jus waiting to be merged.

**Comment by walkawayy** [comment]
> > @ajtudela are you still able to add the Spanish translation? Apologies if you are busy and just haven't got to it yet.
> 
> I made a PR to the branch: [walkawayy#1](https://github.com/walkawayy/TR1X/pull/1)
> 
> Jus waiting to be merged.

Awesome thanks. Sorry about that, I usually don't get PRs on my fork and for some reason Github didn't alert me.

**Comment by walkawayy** [comment]
Sorry need re-reviews to unlock merging. Are we squashing?

**Review by rr- — APPROVED** [review]
Thank you. Yes we definitely squash all into a single commit.

**Review by lahm86 — APPROVED** [review]

**Pull request merged**